Platform/NXP/Sabre_iMX6SX_1GB/Library/iMX6BoardLib/iMX6BoardInit.c

```diff
@@ -0,0 +1,1130 @@
+                              IMX_SPEED_LOW,
+                              IMX_ODE_DISABLE,
+                              IMX_PKE_DISABLE,
+                              IMX_PUE_PULL,
+                              IMX_PUS_100K_OHM_PU,
+                              IMX_HYS_DISABLED,
+                              IMX_SION_DISABLED,
+                              IMX_IOMUXC_ENET1_CRS_ALT5_GPIO2_IO01),
+
+    IMX_PAD_CFG_QSPI1A_DATA2_ALT5_GPIO4_IO18 = _IMX_MAKE_PADCFG(
+                              IMX_SRE_SLOW,
+                              IMX_DSE_260_OHM,
+                              IMX_SPEED_LOW,
+                              IMX_ODE_DISABLE,
+                              IMX_PKE_DISABLE,
+                              IMX_PUE_PULL,
+                              IMX_PUS_100K_OHM_PU,
+                              IMX_HYS_DISABLED,
+                              IMX_SION_DISABLED,
+                              IMX_IOMUXC_QSPI1A_DATA2_ALT5_GPIO4_IO18),
+
+    IMX_PAD_CFG_SD1_DATA1_ALT5_GPIO6_IO03 = _IMX_MAKE_PADCFG(
+                              IMX_SRE_SLOW,
+                              IMX_DSE_260_OHM,
+                              IMX_SPEED_LOW,
+                              IMX_ODE_DISABLE,
+                              IMX_PKE_DISABLE,
+                              IMX_PUE_PULL,
+                              IMX_PUS_100K_OHM_PU,
+                              IMX_HYS_DISABLED,
+                              IMX_SION_DISABLED,
+                              IMX_IOMUXC_SD1_DATA1_ALT5_GPIO6_IO03),
+
+    IMX_PAD_CFG_KEY_COL1_ALT5_GPIO2_IO11 = _IMX_MAKE_PADCFG(
+                              IMX_SRE_SLOW,
+                              IMX_DSE_260_OHM,
+                              IMX_SPEED_LOW,
+                              IMX_ODE_DISABLE,
+                              IMX_PKE_DISABLE,
+                              IMX_PUE_PULL,
+                              IMX_PUS_100K_OHM_PU,
+                              IMX_HYS_DISABLED,
+                              IMX_SION_DISABLED,
+                              IMX_IOMUXC_KEY_COL1_ALT5_GPIO2_IO11)
+} IMX_PWR_PADCFG;
+
+typedef enum {
+    IMX_PAD_CFG_ENET1_COL_GPIO2_IO00 = _IMX_MAKE_PADCFG (
+                              IMX_SRE_SLOW,
+                              IMX_DSE_40_OHM,
+                              IMX_SPEED_MEDIUM,
+                              IMX_ODE_DISABLE,
+                              IMX_PKE_ENABLE,
+                              IMX_PUE_PULL,
+                              IMX_PUS_100K_OHM_PU,
+                              IMX_HYS_ENABLED,
+                              IMX_SION_DISABLED,
+                              IMX_IOMUXC_ENET1_COL_ALT5_GPIO2_IO00),
+} IMX_IMX6Q_PCIE_PADCFG;
+
+/**
+  Initialize SDHC modules on the SOC and perform required pin-muxing
+**/
+VOID SdhcInit ()
+{
+    DebugPrint(0xFFFFFFFF, "VOID SdhcInit()\n");
+
+    //
+    // uSDHC2: SDIO Socket
+    //
+    ImxPadConfig(IMX_PAD_SD2_CLK, IMX_PAD_CFG_SD2_CLK_SD2_CLK);
```

Comment:
Are these already Muxed in U-Boot? If so, please don't mux them again in EDK2.

Applies to all IP. We want to reduce the amount of code in firmware. Typically U-Boot enables specific IP like Sdhc and Enet and muxes the PADs during its init flow. If U-Boot already enabled a specific IP, don't duplicate the code in EDK2.